Header header
string goalID
uint32 width
uint32 height
uint8[] policy